README.md

```diff
@@ -48,11 +66,11 @@ Additional Resources:
 
 ### SAM Stack (template.yaml)
 
-Most components in the developer portal are managed by the SAM stack defined in template.yaml. New application components can be added to this template. Configuration values are fed to this template from the parameter overrides provided on the command line. If overrides are not provided, default values are used.
+All the components in the developer portal are managed by the SAM stack defined in template.yaml. New application components can be added to this template. Configuration values are fed to this template from the parameter overrides provided on the command line. If overrides are not provided, default values are used.
 
 ### UI (/app)
 
-The UI is a simple React application hosted in a public S3 bucket. The client side code communicates with the application backend via an API Gateway proxy API. For more information on updating the UI, see `./dev-portal/README.md`.
+The UI is a simple React application hosted in an S3 bucket. The assets are uploaded to the S3 bucket  The client side code communicates with the application backend via an API Gateway proxy API. For more information on updating the UI, see `./dev-portal/README.md`.
```

Comment:
Missing punctuation after 'The assets are uploaded to the S3 bucket'

Reply:
Went ahead and finished the sentence...